src/sdk/data/classes/token-list-token.ts

```diff
@@ -0,0 +1,13 @@
+export class TokenListToken {
+  address: string;
+
+  chainId: number;
+
+  name: string;
+
+  symbol: string;
+
+  decimals: number;
+
+  logoURI: string;
```

Comment:
What's the reason for the spaces in between each property? The previous class didn't have any so just wondering...

Reply:
Done changes